retrospective-update/functions.py

```
@@ -0,0 +1,733 @@
+                        hourly_zarr: str,
+                        s3_hourly_zarr: str,
+                        credentials: str) -> None:
+    """
+    Check that the following are installed:
+    - awscli
+    - s5cmd
+    - docker
+    """
+    try:
+        subprocess.run(['aws', '--version'], check=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)